### ISSUE TITLE
Training - Animation

### ISSUE THREAD

**Issue by Eonzenex** [hzwer/arXiv2020-RIFE#29]
I'd like to start training a model for animation if you haven't start so already.

Any tips for getting started?
e.g. How should I prep the input data to feed to the train script? Any benefits to using a higher or lower learning rate? How much VRAM a 720p image needs vs 1080p?

**Comment by hzwer** [comment]
I'm working on this. One question I want to know is, does the animation version of DAIN use a special data set or data augment method? I think most of the animation data cannot be directly used for training, due to the discontinuity of painting.

**Comment by a1600012888** [comment]
We feel it is necessary to provide a model specialized for animation data, and we are working on that now

We thought there are two important factors for a model specialized for animation: data & data aug
1. If you have collected your own animation dataset, you can use our trained model as initialization, and fine-tune on your dataset
2. If you don't have an animation dataset, try some color aug, such as colorjittering and https://imgaug.readthedocs.io/en/latest/source/overview/artistic.html#cartoon.  And maybe perceptual loss will help

**Comment by hzwer** [comment]
Compared with v1.1, the v1.4 model greatly reduces the image collapse on 2d animation interpolation.

**Comment by Tsubajashi** [comment]
Thats true, even though the entire beginning of the foot is basically gone.

**Comment by hzwer** [comment]
> Thats true, even though the entire beginning of the foot is basically gone.

v1.5 fix this artifact.

**Comment by Kafu-Renkori** [comment]
I tried interpolating animation with rife-ncnn-vulkan. (I didn't have an NVIDIA GPU, so I couldn't use original RIFE.)
Most animations are 24 fps (25 fps with very rare probability).
So I used rife-ncnn-vulkan twice to make 24 fps to 96 fps and then convert it to 60 fps video.
However, the interpolation performance was not better than that of AMD's frame interpolation technology, FluidMotion(Mode2). (The RIFE was much better in the artifact part.)

The improvement is good in the artifact area, but it seems that there is also a need to improve the screen movement.
This interpolation technology shows better quality than DAIN at the same speed as CAIN.

However, at 30FPS, RIFE is better than FluidMotion (Mode2) in screen movement.

**Comment by Kafu-Yukimura** [comment]
I figured out how to improve the screen movement.
You can check how fluid motion works in Mode 2 and render it similarly through RIFE and FFmpeg.
For example, if you want to make a 24fps video to smooth like Fluid Motion, you can increase the frame multiple of RIFE by x8, and then make it 60fps with the `-framerate 384 -i video.mp4 -vf fps=3840/64` command in ffmpeg.
[framerate (24x8=384), vf fps=(3840÷64=60)]
There is no need to change the structure of RIFE separately to improve screen movement.

This method is much smoother than making the animation at 48fps by setting the frame of the animation to x2 in RIFE, but it can also produce more artifacts.

Video at 25 fps cannot use this method, so we recommend changing to 24 fps and using that method.

**Comment by YiWeiHuang-stack** [comment]
24*8=192
